### PR TITLE
Test

### PR DESCRIPTION
**Pull request by KevinGarvey**
https://apricot-shelbi-11.tiiny.co/

### CODE REVIEW

File: celocations.html [hunk 1]
```diff
@@ -81,21 +81,6 @@ <h2 class="mb-5 font-weight-bold">Fiducial Cosmic Explorer Locations to use for
 
                         </p>
 
-                        <script>
```
Comment (ahnitz) [inline celocations.html:84]: @KevinGarvey I don't think you intend to remove this.

**Comment by ahnitz** [comment]
@KevinGarvey I think we want the links on this page to work. A lot of them are just text. https://apricot-shelbi-11.tiiny.co/article.html?article=JobPostings

**Comment by KevinGarvey** [comment]
That's what I get for trusting that Word knows when I want the link.  updated:  https://apricot-shelbi-11.tiiny.co/

**Review by ahnitz — APPROVED** [review]

**Comment by ahnitz** [comment]
Ok, looks good now

**Comment by KevinGarvey** [comment]
Thanks!

**Pull request merged**